README.md

```diff
@@ -177,6 +177,7 @@ It may contain any of the following:
 
 - `value` - the value that resulted from the get
 - `remainderPath` - If it didn't manage to successfully resolve the whole path through or if simply the `localResolve` option was passed.
+- `cid` - If remainderPath has a value, this will be where it has it's root, otherwise it is the CID that was passed to `.get`
```

Comment:
Is this actually true and if yes, wouldn't it be better if it would otherwise return the last visited CID (so to speak the CID that fully resolved the path)?

Typo: it's => its.

Reply:
It does return the last visited CID, have updated it again.